### PR TITLE
Add unit test for data-hook and instantiateAll

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Add unit test spec for `data-hook.js`.
- Adjust data-hook import to import specific methods.
- Add additional `instantiateAll` test.

## Testing

1. `gulp test:unit` should pass.

### CODE REVIEW

File: test/unit-test/src/cfpb-atomic-component/src/utilities/data-hook-spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  } );
+
+  describe( 'remove()', () => {
+    it( 'should remove a value to the data-* attribute of the element', () => {
+      remove( testComponent, 'test_behavior' );
+      expect( testComponent.getAttribute( 'data-js-hook' ) ).toBe( '' );
+    } );
+  } );
+
+  describe( 'remove()', () => {
```
Comment (contolini) [inline test/unit-test/src/cfpb-atomic-component/src/utilities/data-hook-spec.js:41]: The `remove()` test is in here twice.

File: packages/cfpb-atomic-component/src/utilities/atomic-helpers.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
    - Atom
    ========================================================================= */
 
-import * as dataHook from './data-hook';
+import { add, contains, remove } from './data-hook';
```
Comment (contolini) [inline packages/cfpb-atomic-component/src/utilities/atomic-helpers.js:12]: Why import the individual methods separately? For clarity?

**Review by contolini — APPROVED** [review]

File: test/unit-test/src/cfpb-atomic-component/src/utilities/data-hook-spec.js (hunk 1 above)
Comment (anselmbradford) [reply test/unit-test/src/cfpb-atomic-component/src/utilities/data-hook-spec.js]: 🤔 not sure how that happened, but fixed!

File: packages/cfpb-atomic-component/src/utilities/atomic-helpers.js (hunk 2 above)
Comment (anselmbradford) [reply packages/cfpb-atomic-component/src/utilities/atomic-helpers.js:12]: Yeah so if you search for the methods you will get the import statement in the search results. Also, if data-hook has more added to it, it won't be automatically pulled into this file where it's unneeded.

**Pull request merged**